### ISSUE TITLE
[Inconsistency] Some items do not have new item tooltips

### ISSUE THREAD

**Issue by Aluerie** [ValveSoftware/Dota2-Gameplay#14235]
Continuing #10066 but I think this should be said in a separate issue. 

New Item Tooltips do not cover some items. Mostly, it's passive item effects. Examples of items without new tooltips:

#### Skadi 
* it would be nice to show that its slow doesn't pierce BKB

#### MKB 

* true-strike proc **DOES** pierce BKB 
* the linked damage proc does not pierce BKB (resulting in halved damage)
#### Because unlike Javelin, Maelstorm, Gleipnir, Mjölnir where

* true-strike proc does not pierce BKB
* the linked damage proc does not pierce BKB (but since true-strike never happens - the damage proc also never happens)

---
Also, these tooltips should be linked to item abilities rather than items themselves, because for example, some items have two or more passives/actives involved and some of them should have different information. 

For example, diffusal says

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/33165440/cfb9ae5a-77dd-4f1d-97f2-3cd09190e14b)

which clearly only describes the active component. But what about the fact that Manabreak passive does not pierce BKB. 

---

PS. If I have time I might make somewhat comprehensive list on this but feel free to comment more cases of items that are missing new tooltips or do not have important information listed.

**Comment by rinbega** [comment]
Yeah, Witch Blade has no tooltip at all, Parasma mentions that it is passive and pierces debuff immunity but nothing about being dispellable, damage type, or who it affects, Mage Slayer does not mention debuff immunity[^1].

I also agree that this information should be associated with an ability like on Dota 2 Wiki.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/47411817/77ccc436-07f6-460c-95ec-4096833326ae)

[^1]: Btw. the debuff on enemies still says that it reduces spell damage by 35% and does not say anything about DPS.

**Comment by Aluerie** [comment]
> This issue is stale because it has been open for 365 days with no activity.

All bullet points are still valid. All examples (except Nullifier with debuff immunity) still stand.

**Comment by Aluerie** [comment]
still valid, the tooltips system can be improved.

this stale system is so annoying :D
